### PR TITLE
Adding 2 new Organizations pages and updating 1 page

### PR DESCRIPTION
**Pull request by dunjardl**
New pages:

- Adding a user to an organization
- Creating a team

Updated page:
Can I limit the permissions a user has on a specific project

### CODE REVIEW

**Comment by dunjardl** [comment]
@pyromaniackeca Please review.

File: source/docs/organizations/adding-a-user-to-an-organization.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: Adding a user to an organization
+category: Organizations
+---
+
+Once you have created an organization, you can add users and grant them access
+to different projects. To add an user to an organization, you need to add them
```
Comment (pyromaniackeca) [inline source/docs/organizations/adding-a-user-to-an-organization.md]: an user -> a user

File: source/docs/organizations/adding-a-user-to-an-organization.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+to an existing team, or
+[create a new team within that organization](/docs/organizations/creating-a-team.html).
```
Comment (pyromaniackeca) [inline source/docs/organizations/adding-a-user-to-an-organization.md]: Why is this in a new line?

File: source/docs/organizations/adding-a-user-to-an-organization.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+##Adding a user to a team
+
+Only users who in the _Owners_ team can add other users to different teams. In
```
Comment (pyromaniackeca) [inline source/docs/organizations/adding-a-user-to-an-organization.md]: who in -> who are in

File: source/docs/organizations/adding-a-user-to-an-organization.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+4. Click on the team name.
+
+ <img src="/docs/assets/img/can-i-limit-the-permissions-a-user-has-on-a-specific-project/team-name.png" class="img-responsive img-bordered">
+
+5. Add the user to the team by searching for them by their username
+or the email address in the _Invite by username or email_ field.
+
+ <img src="/docs/assets/img/adding-a-user-to-an-organization/add-user-to-a-team.png" class="img-responsive img-bordered">
+
+ If you invite the new user by email, they will need to accept the invitation
```
Comment (pyromaniackeca) [inline source/docs/organizations/adding-a-user-to-an-organization.md]: I'm not sure this is correct, I think the user only needs to accept if he doesn't have a Semaphore account yet, if he does and you invite him by email I think he's just added. I could be wrong btw.

File: source/docs/organizations/adding-a-user-to-an-organization.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+ before becoming a member of the organization.
+
+##Next steps
+
+Once you have added a user to your organization, you will be able
+to:
```
Comment (pyromaniackeca) [inline source/docs/organizations/adding-a-user-to-an-organization.md]: I think this doesn't need to be a new line.

**Comment by pyromaniackeca** [comment]
It looks fine I would just add that besides giving them access teams also give different permissions.

Also you need to add these pages to the main page.

**Pull request merged**